velox/vector/FlatVector.h

```diff
@@ -403,6 +403,10 @@ class FlatVector final : public SimpleVector<T> {
     return nullptr;
   }
 
+  char* getRawStringBufferWithSpace(vector_size_t /* unused */) {
+    return nullptr;
+  };
```

Comment:
linter pointed out that this semi-colon is not needed; let's remove

I'm also seeing that this method is lacking documentation and tests. Would you submit a separate PR to introduce this method, document it clearly and add a test?